### PR TITLE
Add test to change News post to Guest author

### PR DESCRIPTION
**Pull request by zheileman**
As this test make use of an autocomplete/dropdown selector, a helper has been created to make code more readable, and to reuse the component.

Also, `default_wait_time` is deprecated, changed to `default_max_wait_time`

### CODE REVIEW

File: smoke_tests/happypath/admin_news-post-to-guest-author.feature [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    When I visit "/wp-admin/"
+    # Make sure I am prompted to log in
+    Then I should see "Username or Email Address"
+    # I log in
+    When I log into the intranet as "agency_editor"
+    # I check that I am logged in
+    Then  I should see "Howdy, Agency Editor Test"
+    # Go to an existing post editor page
+    Then I visit "/wp-admin/post.php?post=88805&action=edit"
+    Then I pause for "1" seconds
```
Comment (digitalronin) [inline smoke_tests/happypath/admin_news-post-to-guest-author.feature]: You can use 'And' as a synonym for 'Then', so you get;

Then I do X
And I do Y

...which is a bit more readable for things like pausing.

File: smoke_tests/support/env.rb [hunk 2]
```diff
@@ -11,7 +11,7 @@
 Capybara.run_server = false
 Capybara.default_driver = :poltergeist
 Capybara.javascript_driver = :poltergeist
-Capybara.default_wait_time = 5
+Capybara.default_max_wait_time = 5
```
Comment (tatyree) [inline smoke_tests/support/env.rb]: 👍

File: smoke_tests/support/select_helper.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+module SelectHelper
```
Comment (tatyree) [inline smoke_tests/support/select_helper.rb]: :+1:

File: smoke_tests/happypath/admin_news-post-to-guest-author.feature [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    Then I click the ".ac_match" element
+    # Then I delete the original author
+    Then I click the hidden ".coauthor-row:first-of-type .delete-coauthor" element and accept the confirmation
+    Then I should not see "Shirley-Anne Mullen"
+    Then I should see "Guest Author"
+    Then I click the "Update" button
+    Then I pause for "1" seconds
+    # Let's go check to see if it worked
+    Then I click the "View Post" link
+    Then I should see "Guest Author"
```
Comment (digitalronin) [inline smoke_tests/happypath/admin_news-post-to-guest-author.feature]: I would add here;

`And I should not see "Shirley-Anne Mullen"`

To confirm that you have successfully deleted the original author, and not just added a co-author.

Comment (zheileman) [inline smoke_tests/happypath/admin_news-post-to-guest-author.feature]: Yes, good point. Didn't even know is possible to add co-authors.

**Pull request merged**